recipe/meta.yaml

```diff
@@ -24,12 +24,10 @@ requirements:
     - toolchain  # [unix]
     - python
     - setuptools
-    - numpy x.x
     - numpy >=1.9,<2.0.0
```

Comment:
Can you set `numpy ==1.9` so we build against the minimum available version? Otherwise it will build against the highest available version and users with older versions would be forced to upgrade numpy.

Reply:
Should it be `numpy ==1.9` or `numpy 1.9.*`

Reply:
It is `numpy 1.9.*`.